nncf/telemetry/decorator.py

```diff
@@ -0,0 +1,70 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import inspect
+from typing import Callable
+from typing import List, Union
+
+from nncf.telemetry.events import telemetry_category
+from nncf.telemetry.wrapper import NNCFTelemetry
+from nncf.telemetry.extractors import TelemetryExtractor
+from nncf.telemetry.extractors import VerbatimTelemetryExtractor
+
+
+class tracked_function:
```

Comment:
Please align the class name with the style guide.

Reply:
Does not apply to decorators.

Reply:
need prove:)

Reply:
Surprisingly there are no solid recommendations for that, but all built-in decorators (and external ones too that I've seen) use lower-case naming almost exclusively.

Some discussion on the topic can be found here:
https://stackoverflow.com/questions/10212556/naming-convention-for-descriptors

Reply:
Honestly, I kept in my mind something like that:
```
tracked_function = _TrackedFunction
``